Cargo.toml

```diff
@@ -25,12 +25,12 @@ dusk-jubjub = {version = "0.10.0-rc.0", default-features = false}
 dusk-plonk = {version = "0.8.0-rc.1", default-features = false, features = ["alloc"]}
 num-bigint = {version = "0.3", optional = true } 
 num-traits = {version = "0.2", optional = true } 
-plonk_gadgets = {version = "0.6.0-rc.0", optional = true}
+plonk_gadgets = {version = "0.6.0-rc.0"}
```

Comment:
Replace the `-rc.0` with `-rc`, and take this occasion maybe to do the same for `dusk-plonk` and any other `rc` in this cargo.toml.

Reply:
Already done in https://github.com/dusk-network/dusk-blindbid/pull/132/

Reply:
Then you clearly need to rebase this PR, since you want to merge it with `err_mod_refactor` (#132) and it looks like the code you have here is outdated.